src/stories/atoms/Shape/Shape.style.ts

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+export const ShapeDiv = styled.div<{
+  backgroundColor: string;
+  shapeWidth: string;
+  shapeHeight: string;
+}>`
+  width: ${(props) => props.shapeWidth}px;
```

Comment:
This is an interesting way to concatenate your `shapeWidth` value with `px` but not one that I would recommend. When working with styled-components like this you are setting the value with a immediately invoking function that accepts a `props` argument. This function always returns a string and looks like this:

```ts
// generic <T> is equal to the props that are passed as an argument to `styled` 
((props<T>)) => string
```
I think of it like this:
"Width is equal to the return of my width() function. That return value should be a string that is valid CSS syntax"

So, on to concatenation. If it were me, I'd use a `template literal` to construct that valid `CSS` string:
```ts
{
...rest,
width: ${(props) => `${props.shapeWidth}px`
}
```
The outcome should be the same but this version is more readable and in-line with semantic best practice. 

_Note:_
You might need to tweak that a little bit because of the `${}` that wraps the `(props) => string` function. That's the same syntax you use to access a value inside of a `template literal`  and I'm not 100% sure you can nest those calls.

Reply:
This one I learned a lot of problem solving when it comes to `eslint` errors. Haha, tough battle but well worth it.